pkg/apis/v1/ec2nodeclass_validation.go

```diff
@@ -1,307 +0,0 @@
-/*
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package v1
-
-import (
-	"context"
-	"fmt"
-	"strings"
-
-	"github.com/aws/aws-sdk-go/service/ec2"
-	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
-	"knative.dev/pkg/apis"
-)
-
-const (
```

Comment:
Confirming: we're 100% confident that all of this validation is captured by our current CEL and that there is no runtime validation mechanism performed on this object?

Reply:
I didn't have to rip out any in-code references, so I believe the answer is no. The only place that I could tell this was used was for the webhook. There was runtime validation for the NodePool in the upstream repo, and that was pretty trivially caught by looking for references to the validation functions.

Reply:
I think the ask is to make sure that our coverage in this file is at least a subset of our current CEL validation, right?